libs/utils/android/benchmark.py

```diff
@@ -203,9 +211,15 @@ def __init__(self):
             raise
 
         self._log.info('=== Execution...')
-        self.wl.run(out_dir=self.out_dir,
-                    collect=self._getBmCollect(),
-                    **self.bm_params)
+        for iter in range(self.bm_iterations):
```

Comment:
what about:

```python
for iter_id in range(1, self.bm_iterations+1):
```

So that we get more humans iterations ID which starts from 1... and we avoid the small +1 in the following loop?